docker-compose.yml

```diff
@@ -32,23 +32,23 @@ services:
 
   nginx:
     build: nginx
-    env_file: .env
+    env_file: ${PROJECT_LOCATION}/docker/local/.env
     ports:
       - 443:443
     volumes:
-      - ${DOCKER_MOUNT_POINT}:/var/www/html:rw,delegated
+      - ${PROJECT_LOCATION}:/var/www/html:rw,delegated
       # Custom configuration
-      - ./nginx/servers/magento.conf:/etc/nginx/conf.d/magento.conf:ro
-      - ./nginx/servers/custom.conf:/etc/nginx/conf.d/custom.conf:ro
+      - ./nginx/servers/upstream.conf:/etc/nginx/conf.d/upstream.conf:ro
```

Comment:
Is it really useful to split upstream and vhost if you keep only one custom conf file ?

Reply:
Good point. 😄